examples/get_project_by_id.go

```diff
@@ -69,6 +69,7 @@ func main() {
 		Proxy(proxy).
 		CAFile(caFile).
 		Insecure(insecure).
+		Debug(true).
```

Comment:
In all the examples you'll need to remove `-logtostderr...` and `-v..` (see file head)
It would be nice if you can add a debug var to init and print according to that

Reply:
Done.